app/controllers/procedures_controller.rb

```diff
@@ -71,7 +75,22 @@ def update
   end
 
   def destroy
+    @appointment = @procedure.appointment
+    @statuses = @appointment.appointment_statuses.map{|x| x.status}
```

Comment:
All of these `.map{|x| x.status}` can be switched to `.pluck`

Reply:
Oh yeah, probably. That's existing code I copied from other controller actions.